src/azul/plugins/metadata/hca/transform.py

```diff
@@ -342,22 +492,34 @@ def _donor_types(cls) -> FieldTypes:
             'biological_sex': null_str,
             'genus_species': null_str,
             'diseases': null_str,
-            'organism_age': null_str,
+            'organism_age': value_and_unit,
             'organism_age_unit': null_str,
+            'organism_age_value': null_str,
             # Prevent problem due to shadow copies on numeric ranges
             'organism_age_range': pass_thru_json,
             # Pass through field added by DonorOrganismAggregator
             'donor_count': pass_thru_int
         }
 
     def _donor(self, donor: api.DonorOrganism) -> MutableJSON:
+        if donor.organism_age is None:
+            if donor.organism_age_unit is None:
+                organism_age = None
+            else:
+                raise Exception
```

Comment:
Exception is a bit generic. I think we should convert this if-else to a use of`require`.